// unstructured-ds/dist/build/tokens.js
const e = "#f0fbf4", s = "#e9f7e9", c = "#d1f0d8", n = "#b8e6c6", t = "#9eddb4", o = "#7fd3a1", a = "#5ec98e", r = "#3cbf79", f = "#029a34", d = "#028c30", l = "#027e2c", S = "#026f28", u = "#025f23", i = "#014f1d", G = "#013f17", y = "#002f11", B = "#f0fbf433", b = "#e9f7e933", g = "#d1f0d833", I = "#b8e6c633", U = "#9eddb433", T = "#7fd3a133", R = "#5ec98e33", D = "#3cbf7933", x = "#029a3433", v = "#028c3033", w = "#027e2c33", O = "#026f2833", Y = "#025f2333", H = "#014f1d33", F = "#013f1733", L = "#002f1133", h = "#fff0f3", k = "#ffe6ea", z = "#ffd0d7", p = "#ffb8c2", P = "#ff9fab", W = "#ff8695", m = "#ff6b82", C = "#ff4f65", E = "#f00038", M = "#d60032", X = "#bd002c", N = "#a30026", A = "#890020", Q = "#6f001a", j = "#550014", q = "#3b000e", J = "#fff0f333", K = "#ffe6ea33", V = "#ffd0d733", Z = "#ffb8c233", _ = "#ff9fab33", $ = "#ff869533", ee = "#ff6b8233", se = "#ff4f6533", ce = "#f0003833", ne = "#d6003233", te = "#bd002c33", oe = "#a3002633", ae = "#89002033", re = "#6f001a33", fe = "#55001433", de = "#3b000e33", le = "#fff9f0", Se = "#fff1dc", ue = "#ffe5bd", ie = "#ffd799", Ge = "#ffc775", ye = "#ffb652", Be = "#ffa733", be = "#ff9814", ge = "#fa9200", Ie = "#e88300", Ue = "#d67500", Te = "#c46700", Re = "#a85500", De = "#8c4400", xe = "#703300", ve = "#552400", we = "#fff9f033", Oe = "#fff1dc33", Ye = "#ffe5bd33", He = "#ffd79933", Fe = "#ffc77533", Le = "#ffb65233", he = "#ffa73333", ke = "#ff981433", ze = "#fa920033", pe = "#e8830033", Pe = "#d6750033", We = "#c4670033", me = "#a8550033", Ce = "#8c440033", Ee = "#70330033", Me = "#55240033", Xe = "#fff8f3", Ne = "#ffefe3", Ae = "#ffe3cc", Qe = "#ffd1b0", je = "#ffbc8c", qe = "#ffa866", Je = "#ff9140", Ke = "#ff7320", Ve = "#ff5b00", Ze = "#e65300", _e = "#cc4900", $e = "#b23f00", es = "#993600", ss = "#802c00", cs = "#662200", ns = "#4c1900", ts = "#fff8f333", os = "#ffefe333", as = "#ffe3cc33", rs = "#ffd1b033", fs = "#ffbc8c33", ds = "#ffa86633", ls = "#ff914033", Ss = "#ff732033", us = "#ff5b0033", is = "#e6530033", Gs = "#cc490033", ys = "#b23f0033", Bs = "#99360033", bs = "#802c0033", gs = "#66220033", Is = "#4c190033", Us = "#ffffff", Ts = "#fafafa", Rs = "#f4f4f4", Ds = "#eeeeee", xs = "#e5e5e5", vs = "#d9d9d9", ws = "#c9c9c9", Os = "#b5b5b5", Ys = "#a0a0a0", Hs = "#8a8a8a", Fs = "#747474", Ls = "#5e5e5e", hs = "#424242", ks = "#2c2c2c", zs = "#191919", ps = "#141414", Ps = "#ffffff33", Ws = "#fafafa33", ms = "#f4f4f433", Cs = "#eeeeee33", Es = "#e5e5e533", Ms = "#d9d9d933", Xs = "#c9c9c933", Ns = "#b5b5b533", As = "#a0a0a033", Qs = "#8a8a8a33", js = "#74747433", qs = "#5e5e5e33", Js = "#42424233", Ks = "#2c2c2c33", Vs = "#19191933", Zs = "#14141433", _s = 0, $s = 2, ec = 4, sc = 6, cc = 8, nc = 10, tc = 12, oc = 14, ac = 16, rc = 18, fc = 20, dc = 22, lc = 24, Sc = 26, uc = 28, ic = 30, Gc = 32, yc = 34, Bc = 36, bc = 38, gc = 40, Ic = 42, Uc = 44, Tc = 46, Rc = 48, Dc = 50, xc = 52, vc = 54, wc = 56, Oc = 58, Yc = 60, Hc = 62, Fc = 64, Lc = 66, hc = 68, kc = 70, zc = 72, pc = 74, Pc = 76, Wc = 78, mc = 80, Cc = 82, Ec = 84, Mc = 86, Xc = 88, Nc = 90, Ac = 92, Qc = 94, jc = 96, qc = 98, Jc = 100, Kc = 250, Vc = "#fff8f3", Zc = "#ffefe3", _c = "#ffe3cc", $c = "#ffd1b0", en = "#ffbc8c", sn = "#ffa866", cn = "#ff9140", nn = "#ff7320", tn = "#ff5b00", on = "#e65300", an = "#cc4900", rn = "#b23f00", fn = "#993600", dn = "#802c00", ln = "#662200", Sn = "#4c1900", un = "#fff8f333", Gn = "#ffefe333", yn = "#ffe3cc33", Bn = "#ffd1b033", bn = "#ffbc8c33", gn = "#ffa86633", In = "#ff914033", Un = "#ff732033", Tn = "#ff5b0033", Rn = "#e6530033", Dn = "#cc490033", xn = "#b23f0033", vn = "#99360033", wn = "#802c0033", On = "#66220033", Yn = "#4c190033", Hn = "#f4f4f4", Fn = "#f4f4f4", Ln = "#f4f4f4", hn = "#e5e5e5", kn = "#eeeeee", zn = "#ffd1b0", pn = "#fff8f3", Pn = "#ffbc8c", Wn = "#ffefe3", mn = "#fff8f3", Cn = "#026f28", En = "#029a34", Mn = "#e9f7e9", Xn = "#a30026", Nn = "#f00038", An = "#ffe6ea", Qn = "#c46700", jn = "#fa9200", qn = "#fff1dc", Jn = "#fafafa", Kn = "#eeeeee", Vn = "#e5e5e5", Zn = "#d9d9d9", _n = "#eeeeee", $n = "#d9d9d9", et = "#e5e5e5", st = "#eeeeee", ct = "#c9c9c9", nt = "#e5e5e5", tt = "#ffffff", ot = "#ffe3cc", at = "#ffefe3", rt = "#ffa866", ft = "#ffbc8c", dt = "#ffefe3", lt = "#ff5b00", St = "#ffa866", ut = "#802c00", it = "#b23f00", Gt = "#ff5b00", yt = "#141414", Bt = "#2c2c2c", bt = "#424242", gt = "#747474", It = "#5e5e5e", Ut = "#191919", Tt = "#424242", Rt = "#e5e5e5", Dt = "#5e5e5e", xt = "#2c2c2c", vt = "#747474", wt = "#8a8a8a", Ot = "#5e5e5e", Yt = "#424242", Ht = "#8a8a8a", Ft = "#a0a0a0", Lt = "#747474", ht = "#5e5e5e", kt = "#b23f00", zt = "#141414", pt = "#2c2c2c", Pt = "#5e5e5e", Wt = "#3b000e", mt = "#ff5b00", Ct = "#552400", Et = "#002f11", Mt = "#4c1900", Xt = "#8a8a8a", Nt = "#a0a0a0", At = "#b5b5b5", Qt = "#eeeeee", jt = "#d9d9d9", qt = "#c9c9c9", Jt = "#ffe3cc", Kt = "#f00038", Vt = "#029a34", Zt = "#fa9200", _t = "#d9d9d9", $t = "#e65300", eo = "#d9d9d9", so = "#d9d9d9", co = "#e65300", no = "#141414", to = "#ff5b00", oo = "#e65300", ao = "#029a34", ro = "#f00038", fo = "#fa9200", lo = "#424242", So = "#141414", uo = "#a0a0a0", io = "#ffffff", Go = "#f4f4f4", yo = "#424242", Bo = "#026f28", bo = "#a30026", go = "#c46700", Io = 0, Uo = 4, To = 6, Ro = 8, Do = 12, xo = 16, vo = 24, wo = 32, Oo = 80, Yo = 250, Ho = "#ff5b00", Fo = "#ff5b0033", Lo = 2, ho = "Regular", ko = "Medium", zo = "Bold", po = "11rem", Po = 12, Wo = 14, mo = 16, Co = 18, Eo = 24, Mo = 32, Xo = 40, No = 48, Ao = 56, Qo = 64, jo = 88, qo = 14, Jo = 16, Ko = 20, Vo = 24, Zo = 32, _o = 36, $o = 42, ea = 48, sa = 58, ca = 62, na = 64, ta = 84, oa = 0, aa = -0.20000000298023224, ra = -0.4000000059604645, fa = -0.6000000238418579, da = -1, la = -2, Sa = "Satoshi", ua = "Satoshi Display";
export {
  Io as BorderRadius0,
  Do as BorderRadius12,
  xo as BorderRadius16,
  vo as BorderRadius24,
  wo as BorderRadius32,
  Uo as BorderRadius4,
  To as BorderRadius6,
  Ro as BorderRadius8,
  Oo as BorderRadius80,
  Yo as BorderRadiusFull,
  Jt as BorderStrokeBrand,
  $t as BorderStrokeBrandActive,
  co as BorderStrokeBrandActive2,
  jt as BorderStrokeDefault,
  _t as BorderStrokeDisabled,
  Kt as BorderStrokeError,
  eo as BorderStrokeHover,
  qt as BorderStrokeMedium,
  so as BorderStrokePressed,
  Qt as BorderStrokeSubtle,
  Vt as BorderStrokeSuccess,
  Zt as BorderStrokeWarning,
  Vc as Brand1,
  on as Brand10,
  an as Brand11,
  rn as Brand12,
  fn as Brand13,
  dn as Brand14,
  ln as Brand15,
  Sn as Brand16,
  Zc as Brand2,
  _c as Brand3,
  $c as Brand4,
  en as Brand5,
  sn as Brand6,
  cn as Brand7,
  nn as Brand8,
  tn as Brand9,
  un as BrandGlass1,
  Rn as BrandGlass10,
  Dn as BrandGlass11,
  xn as BrandGlass12,
  vn as BrandGlass13,
  wn as BrandGlass14,
  On as BrandGlass15,
  Yn as BrandGlass16,
  Gn as BrandGlass2,
  yn as BrandGlass3,
  Bn as BrandGlass4,
  bn as BrandGlass5,
  gn as BrandGlass6,
  In as BrandGlass7,
  Un as BrandGlass8,
  Tn as BrandGlass9,
  no as DropshadowSuccessDefault,
  ua as FontFamilyDisplay,
  Sa as FontFamilySans,
  Eo as FontSize2xl,
  Mo as FontSize3xl,
  Xo as FontSize4xl,
  No as FontSize5xl,
  Ao as FontSize6xl,
  Qo as FontSize7xl,
  jo as FontSize8xl,
  mo as FontSizeL,
  Wo as FontSizeM,
  Po as FontSizeS,
  Co as FontSizeXl,
  po as FontSizeXs,
  ho as FontWeight400,
  ko as FontWeight500,
  zo as FontWeight700,
  e as Green1,
  d as Green10,
  l as Green11,
  S as Green12,
  u as Green13,
  i as Green14,
  G as Green15,
  y as Green16,
  s as Green2,
  c as Green3,
  n as Green4,
  t as Green5,
  o as Green6,
  a as Green7,
  r as Green8,
  f as Green9,
  B as GreenGlass1,
  v as GreenGlass10,
  w as GreenGlass11,
  O as GreenGlass12,
  Y as GreenGlass13,
  H as GreenGlass14,
  F as GreenGlass15,
  L as GreenGlass16,
  b as GreenGlass2,
  g as GreenGlass3,
  I as GreenGlass4,
  U as GreenGlass5,
  T as GreenGlass6,
  R as GreenGlass7,
  D as GreenGlass8,
  x as GreenGlass9,
  Us as Grey1,
  Hs as Grey10,
  Fs as Grey11,
  Ls as Grey12,
  hs as Grey13,
  ks as Grey14,
  zs as Grey15,
  ps as Grey16,
  Ts as Grey2,
  Rs as Grey3,
  Ds as Grey4,
  xs as Grey5,
  vs as Grey6,
  ws as Grey7,
  Os as Grey8,
  Ys as Grey9,
  Ps as GreyGlass1,
  Qs as GreyGlass10,
  js as GreyGlass11,
  qs as GreyGlass12,
  Js as GreyGlass13,
  Ks as GreyGlass14,
  Vs as GreyGlass15,
  Zs as GreyGlass16,
  Ws as GreyGlass2,
  ms as GreyGlass3,
  Cs as GreyGlass4,
  Es as GreyGlass5,
  Ms as GreyGlass6,
  Xs as GreyGlass7,
  Ns as GreyGlass8,
  As as GreyGlass9,
  to as IconIconBrandDefault,
  oo as IconIconBrandSecondary,
  fo as IconIconCautionDefault,
  go as IconIconCautionSecondary,
  ro as IconIconCriticalDefault,
  bo as IconIconCriticalSecondary,
  lo as IconIconDefault,
  uo as IconIconDisabled,
  io as IconIconInverseDefault,
  Go as IconIconInverseSecondary,
  yo as IconIconInverseTertiary,
  So as IconIconSecondary,
  ao as IconIconSuccessDefault,
  Bo as IconIconSuccessSecondary,
  Ho as IconbuttonIconbuttonBackground,
  Lo as IconbuttonIconbuttonBorderRadius,
  Fo as IconbuttonIconbuttonSurface,
  oa as LetterSpacingBaggy,
  ra as LetterSpacingComfy,
  fa as LetterSpacingFitted,
  aa as LetterSpacingLoose,
  da as LetterSpacingSnug,
  la as LetterSpacingTight,
  _o as LineHeight2xl,
  $o as LineHeight3xl,
  ea as LineHeight4xl,
  sa as LineHeight5xl,
  ca as LineHeight6xl,
  na as LineHeight7xl,
  ta as LineHeight8xl,
  Vo as LineHeightL,
  Ko as LineHeightM,
  Jo as LineHeightS,
  Zo as LineHeightXl,
  qo as LineHeightXs,
  Xe as Orange1,
  Ze as Orange10,
  _e as Orange11,
  $e as Orange12,
  es as Orange13,
  ss as Orange14,
  cs as Orange15,
  ns as Orange16,
  Ne as Orange2,
  Ae as Orange3,
  Qe as Orange4,
  je as Orange5,
  qe as Orange6,
  Je as Orange7,
  Ke as Orange8,
  Ve as Orange9,
  ts as OrangeGlass1,
  is as OrangeGlass10,
  Gs as OrangeGlass11,
  ys as OrangeGlass12,
  Bs as OrangeGlass13,
  bs as OrangeGlass14,
  gs as OrangeGlass15,
  Is as OrangeGlass16,
  os as OrangeGlass2,
  as as OrangeGlass3,
  rs as OrangeGlass4,
  fs as OrangeGlass5,
  ds as OrangeGlass6,
  ls as OrangeGlass7,
  Ss as OrangeGlass8,
  us as OrangeGlass9,
  h as Red1,
  M as Red10,
  X as Red11,
  N as Red12,
  A as Red13,
  Q as Red14,
  j as Red15,
  q as Red16,
  k as Red2,
  z as Red3,
  p as Red4,
  P as Red5,
  W as Red6,
  m as Red7,
  C as Red8,
  E as Red9,
  J as RedGlass1,
  ne as RedGlass10,
  te as RedGlass11,
  oe as RedGlass12,
  ae as RedGlass13,
  re as RedGlass14,
  fe as RedGlass15,
  de as RedGlass16,
  K as RedGlass2,
  V as RedGlass3,
  Z as RedGlass4,
  _ as RedGlass5,
  $ as RedGlass6,
  ee as RedGlass7,
  se as RedGlass8,
  ce as RedGlass9,
  Hn as SurfaceSurfaceBackground,
  mn as SurfaceSurfaceBrandDefault,
  zn as SurfaceSurfaceBrandDisabled,
  Wn as SurfaceSurfaceBrandHover,
  Pn as SurfaceSurfaceBrandPressed,
  kt as SurfaceSurfaceBrandQuaternaryDefault,
  at as SurfaceSurfaceBrandSecondaryDefault,
  ft as SurfaceSurfaceBrandSecondaryDisabled,
  ot as SurfaceSurfaceBrandSecondaryHover,
  rt as SurfaceSurfaceBrandSecondaryPressed,
  dt as SurfaceSurfaceBrandSecondarySelected,
  pn as SurfaceSurfaceBrandSelected,
  Gt as SurfaceSurfaceBrandTertiaryDefault,
  St as SurfaceSurfaceBrandTertiaryDisabled,
  it as SurfaceSurfaceBrandTertiaryHover,
  ut as SurfaceSurfaceBrandTertiaryPressed,
  lt as SurfaceSurfaceBrandTertiarySelected,
  tt as SurfaceSurfaceDefault,
  Ln as SurfaceSurfaceDisabled,
  An as SurfaceSurfaceErrorDefault,
  Nn as SurfaceSurfaceErrorSecondary,
  Xn as SurfaceSurfaceErrorTertiary,
  Fn as SurfaceSurfaceHover,
  yt as SurfaceSurfaceInverseDefault,
  gt as SurfaceSurfaceInverseDisabled,
  Bt as SurfaceSurfaceInverseHover,
  bt as SurfaceSurfaceInversePressed,
  Ut as SurfaceSurfaceInverseSecondaryDefault,
  wt as SurfaceSurfaceInverseSecondaryDisabled,
  Yt as SurfaceSurfaceInverseSecondaryHover,
  Ot as SurfaceSurfaceInverseSecondaryPressed,
  vt as SurfaceSurfaceInverseSecondarySelected,
  It as SurfaceSurfaceInverseSelected,
  Tt as SurfaceSurfaceInverseTertiaryDefault,
  Ft as SurfaceSurfaceInverseTertiaryDisabled,
  ht as SurfaceSurfaceInverseTertiaryHover,
  Lt as SurfaceSurfaceInverseTertiaryPressed,
  Ht as SurfaceSurfaceInverseTertiarySelected,
  Rt as SurfaceSurfaceNeutralDefault,
  Dt as SurfaceSurfaceNeutralSecondary,
  xt as SurfaceSurfaceNeutralTertiary,
  hn as SurfaceSurfacePressed,
  Jn as SurfaceSurfaceSecondaryDefault,
  Kn as SurfaceSurfaceSecondaryDisabled,
  _n as SurfaceSurfaceSecondaryHover,
  Zn as SurfaceSurfaceSecondaryPressed,
  Vn as SurfaceSurfaceSecondarySelected,
  kn as SurfaceSurfaceSelected,
  Mn as SurfaceSurfaceSuccessDefault,
  En as SurfaceSurfaceSuccessSecondary,
  Cn as SurfaceSurfaceSuccessTertiary,
  st as SurfaceSurfaceTertiaryDefault,
  nt as SurfaceSurfaceTertiaryDisabled,
  et as SurfaceSurfaceTertiaryHover,
  ct as SurfaceSurfaceTertiaryPressed,
  $n as SurfaceSurfaceTertiarySelected,
  qn as SurfaceSurfaceWarningDefault,
  jn as SurfaceSurfaceWarningSecondary,
  Qn as SurfaceSurfaceWarningTertiary,
  mt as TextTextBrandDefault,
  Mt as TextTextBrandSecondary,
  zt as TextTextDefault,
  Wt as TextTextErrorDefault,
  Xt as TextTextInverseDefault,
  Nt as TextTextInverseSecondary,
  At as TextTextInverseTertiary,
  pt as TextTextSecondary,
  Et as TextTextSuccessDefault,
  Pt as TextTextTertiary,
  Ct as TextTextWarningDefault,
  _s as Units0,
  nc as Units10,
  Jc as Units100,
  tc as Units12,
  oc as Units14,
  ac as Units16,
  rc as Units18,
  $s as Units2,
  fc as Units20,
  dc as Units22,
  lc as Units24,
  Kc as Units250,
  Sc as Units26,
  uc as Units28,
  ic as Units30,
  Gc as Units32,
  yc as Units34,
  Bc as Units36,
  bc as Units38,
  ec as Units4,
  gc as Units40,
  Ic as Units42,
  Uc as Units44,
  Tc as Units46,
  Rc as Units48,
  Dc as Units50,
  xc as Units52,
  vc as Units54,
  wc as Units56,
  Oc as Units58,
  sc as Units6,
  Yc as Units60,
  Hc as Units62,
  Fc as Units64,
  Lc as Units66,
  hc as Units68,
  kc as Units70,
  zc as Units72,
  pc as Units74,
  Pc as Units76,
  Wc as Units78,
  cc as Units8,
  mc as Units80,
  Cc as Units82,
  Ec as Units84,
  Mc as Units86,
  Xc as Units88,
  Nc as Units90,
  Ac as Units92,
  Qc as Units94,
  jc as Units96,
  qc as Units98,
  le as Yellow1,
  Ie as Yellow10,
  Ue as Yellow11,
  Te as Yellow12,
  Re as Yellow13,
  De as Yellow14,
  xe as Yellow15,
  ve as Yellow16,
  Se as Yellow2,
  ue as Yellow3,
  ie as Yellow4,
  Ge as Yellow5,
  ye as Yellow6,
  Be as Yellow7,
  be as Yellow8,
  ge as Yellow9,
  we as YellowGlass1,
  pe as YellowGlass10,
  Pe as YellowGlass11,
  We as YellowGlass12,
  me as YellowGlass13,
  Ce as YellowGlass14,
  Ee as YellowGlass15,
  Me as YellowGlass16,
  Oe as YellowGlass2,
  Ye as YellowGlass3,
  He as YellowGlass4,
  Fe as YellowGlass5,
  Le as YellowGlass6,
  he as YellowGlass7,
  ke as YellowGlass8,
  ze as YellowGlass9
};
